app/src/main/java/nsit/app/com/nsitapp/CalculatorActivity.java

```diff
@@ -14,14 +18,27 @@
 import android.widget.TextView;
 import android.widget.Toast;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import static nsit.app.com.nsitapp.R.id.per;
+
 public class CalculatorActivity extends Fragment {
 
     private Spinner s1,s2;
-    private TextView tc10, tc11, tc12, tc9, tc4, tc5;
+    private TextView tc10, tc11, tc12, tc9, tc4, tc5, tc1,tc2,tc3,tc8,tc7,tc6,rs;
+    private double n1=-1.0, n2=-1.0, n3=-1.0, n4=-1.0, n5=-1.0, n6=-1.0, n7=-1.0, n8=-1.0, n9=-1.0, n10=-1.0, n11=-1.0, n12=-1.0;
+    private EditText t1,t2,t3,t4,t5,t6,t7,t8,t9,t10,t11,t12;
+
```

Comment:
Change the name of all the variables to something appropriate. Something these variables signify.